### ISSUE TITLE
Log of things which might always require interactive debugging on live data

### ISSUE THREAD

**Issue by sebbacon** [opensafely/cohort-extractor#212]
* dealing with the [huge plots issue](https://github.com/opensafely/risk-factors-research/issues/13) that we had early on in the RF study; 
* getting mortality plots on a useful scale;  
* model convergence issues in Fizz's risk prediction work where complex models used

**Comment by ghickman** [comment]
> import pdb; pdb.set_trace()

We have [`breakpoint()`](https://www.python.org/dev/peps/pep-0553/) since 3.7!  Which is exactly the same when called without arguments, but ties in nicely to the `opensafely breakpoint` subcommand from a marketing/docs point of view.

**Comment by HelenCEBM** [comment]
This relates to #556. 

Any updates?

**Comment by sebbacon** [comment]
We we probably be starting work on addressing at least some of this in a couple of weeks

**Comment by sebbacon** [comment]
I agree. The proposal was really about debugging problems, whether they're graphs or something else.  Could be spinning data shaping scripts, for example.
